### PR TITLE
[SPARK-13153][PySpark] ML persistence failed when handle no default value parameter

### PR DESCRIPTION
**Pull request by Wenpei**
Fix this defect by check default value exist or not.

@yanboliang Please help to review.

### CODE REVIEW

File: python/pyspark/ml/wrapper.py [hunk 1]
```diff
@@ -79,8 +79,9 @@ def _transfer_params_from_java(self):
         for param in self.params:
             if self._java_obj.hasParam(param.name):
                 java_param = self._java_obj.getParam(param.name)
-                value = _java2py(sc, self._java_obj.getOrDefault(java_param))
-                self._paramMap[param] = value
+                if self._java_obj.hasDefault(java_param):
```
Comment (yanboliang) [inline python/pyspark/ml/wrapper.py]: We should check by ```isDefined``` rather than ```hasDefault``` here.

**Comment by yanboliang** [comment]
ping @mengxr @jkbradley 
Could you add @Wenpei to white list ? This is an obvious bug and we should fix it.

**Comment by Wenpei** [comment]
It's OK now and ready to launch test.

**Comment by mengxr** [comment]
ok to test

**Comment by mengxr** [comment]
LGTM pending Jenins

**Comment by mengxr** [comment]
Merged into master and branch-1.6. Thanks!

**Pull request closed without merging**